plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -386,17 +445,30 @@ function OPDSBrowser:genItemTableFromCatalog(catalog, item_url)
                 end
                 if link.type:find(self.search_type) then
                     if link.href then
-                        table.insert(item_table, { -- the first item in each subcatalog
-                            text       = "\u{f002} " .. _("Search"), -- append SEARCH icon
```

Comment:
This is why the test fails, the text string wasn't restored as-is ;).

Reply:
(i.e., at a very quick glance, it looks like you might have reverted some of the refactoring changes?).

Reply:
So Weird! I don't remember even touching that part. I'll put it back.

Reply:
Fixed in Latest. Yeah, somehow I managed to _not_ pull  the correct opdsbrowser before I even started working on this Branch, pulled latest version, replaced it, working now.